### PR TITLE
Prereqs and docs for setting up the Postgres Debezium connector

### PR DESCRIPTION
**Pull request by DanRoseus**
I've completed an e2e test of this, installed using this workspace:
https://cloud.ibm.com/schematics/workspaces/ibm-cp-integration-09-03-2020-dan-debezium-e2e-69a1f942-56ce-4e/log/00fc8090e31d42dc34e8a2586807657d?region=us

Resulting in the following events in ES:
image

### CODE REVIEW

File: EventEnabledInsurance/prereqs.sh [hunk 1]
```diff
@@ -132,6 +132,35 @@ fi
 
 echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
+REPLICATION_USER=$(echo ${namespace}_sor_reeplication_${SUFFIX} | sed 's/-/_/g')
```
Comment (laurensks) [inline EventEnabledInsurance/prereqs.sh:135]: I think that's a spelling mistake there?
```suggestion
REPLICATION_USER=$(echo ${namespace}_sor_replication_${SUFFIX} | sed 's/-/_/g')
```

File: EventEnabledInsurance/prereqs.sh [hunk 2]
```diff
@@ -132,6 +132,35 @@ fi
...
+REPLICATION_PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 32 ; echo)
+REPLICATION_PASSWORD_ENCODED=$(echo -n ${REPLICATION_PASSWORD} | base64)
+
+DB_POD=$(oc get pod -n ${POSTGRES_NAMESPACE} -l name=postgresql -o jsonpath='{.items[].metadata.name}')
```
Comment (laurensks) [inline EventEnabledInsurance/prereqs.sh:139]: The DB_POD is on line 77 as well. Is this a different pod to that one or can it be removed?

Comment (DanRoseus) [reply EventEnabledInsurance/prereqs.sh]: Good point, nothing has changed to restart the DB_POD. Those lines have been removed

**Review by laurensks — APPROVED** [review]
looks good! Nice!

**Pull request merged**